logspout/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM gliderlabs/logspout:v3.2.3
```

Comment:
I'd put the logspout version in ```build.yml```. That would basically allow us to have different versions of ```monasca/logsput``` using potentially different versions of ```logsput``` without the need of changing or having multiple Dockerfiles.

Reply:
Done